From 2086bb077a5ef278fb159a33eb46fca5ab7804a2 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Wed, 24 Nov 2021 01:01:39 +0100
Subject: [PATCH 0224/1695] plat-stm32mp1: conf: force using firewall framework

Update the platform to use the firewall API in case
of ETZPC is enabled.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f7167df491afa3b03f16f6df1cab7d941ba1a8d2 plat-stm32mp1: conf: force using firewall framework]
Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I5d3cf4c4e0096519145d93a274a882cae9408599
---
 core/arch/arm/plat-stm32mp1/conf.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index bb124703d..def49aa28 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -264,6 +264,10 @@ ifneq ($(CFG_WITH_SOFTWARE_PRNG),y)
 $(call force,CFG_STM32_RNG,y,Required by HW RNG when CFG_WITH_SOFTWARE_PRNG=n)
 endif
 
+ifeq ($(CFG_STM32_ETZPC),y)
+$(call force,CFG_STM32_FIREWALL,y)
+endif
+
 ifeq ($(CFG_STPMIC1),y)
 $(call force,CFG_STM32_I2C,y)
 $(call force,CFG_STM32_GPIO,y)
-- 
2.47.2

